samples/SwiftDemoApp/SwiftDemoApp/MasterViewController.swift

```diff
@@ -32,20 +33,21 @@ class MasterViewController: UITableViewController {
 
   override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) ->
       UITableViewCell {
-    let cellIdentifier = "Cell"
-    var cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier)
-       
-    if (cell == nil) {
-      cell = UITableViewCell(style: .subtitle, reuseIdentifier: cellIdentifier)
-      cell!.accessoryType = .disclosureIndicator
-    }
-
+        
+    let cell: UITableViewCell = {
+    guard let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier) else {
+        return UITableViewCell(style: .subtitle, reuseIdentifier: cellIdentifier)
+        }
+        return cell
+    }()
```

Comment:
This can be simplified to:

```swift
guard let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier) else {
        return UITableViewCell(style: .subtitle, reuseIdentifier: cellIdentifier)
}
```

Reply:
This will not help. As we are using default cell Instead of Custom Cell. Please find the screenshot.
<img width="1429" alt="Screenshot 2020-07-10 at 10 03 18 PM" src="https://user-images.githubusercontent.com/8708394/87177516-6546b780-c2f9-11ea-85fc-ff1b016cb660.png">

Reply:
Ah, I think `guard` is not the right construct here then since `guard` is for early returns. How about:

```swift
let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier) ?? 
   UITableViewCell(style: .subtitle, reuseIdentifier: cellIdentifier)
}
```

Reply:
Yeah, @arriolac That will work for sure. How I have implemented is a kind of closure. But now the choice is yours. Both the approach will work. Tell me which one to do.

Reply:
I think the nil coalescing operator (double question mark) syntax that I suggested is a bit more readable. Please change it to that and we can go ahead and merge this.

Reply:
@arriolac done! 💯